alfalfa_web/server/resolvers.js

```diff
@@ -83,11 +83,11 @@ function runSiteResolver(args, context) {
         "job": "${job}",
         "params": {
           "run_id": "${args.siteRef}",
-          "timescale": ${args.timescale},
+          "timescale": "${args.timescale}",
           "start_datetime": "${args.startDatetime}",
           "end_datetime": "${args.endDatetime}",
-          "realtime": ${args.realtime},
-          "external_clock": ${args.externalClock}
+          "realtime": "${args.realtime}",
+          "external_clock": "${args.externalClock}"
         }
```

Comment:
Do we want the same `undefined` protections in this as in the `api-v2.js`?

Reply:
oh, with the `!!` and `|| 5`. If so, then yeah, we should be consistent.